charts/ark-pentaho-ce/files/server.properties

```diff
@@ -0,0 +1,33 @@
+# FullyQualifiedServerUrl is used only in the case of offline content generation
+# and whenever something need to talk back to the server
+#fully-qualified-server-url=http://localhost:2002/pentaho
+fully-qualified-server-url=http://localhost:2002/pentaho

```

Comment:
Please use only LF for line ending (UNIX style), rather than CRLF (Windows style). Ditto for the rest of the file.

Reply:
thanks for spotting this out changed to LF.